### PR TITLE
Adapt from template to setup workshop website

### PR DESCRIPTION
**Pull request by svenvanderburg**
To do:
* Add git quiz (@svenvanderburg will add it in a separate PR)
* Collaborative document link (@mkuzak you mentioned we could easily refer to a hack md?)

### CODE REVIEW

File: _includes/cr/syllabus.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+<div class="row">
+    <div class="col-md-6">
+    <h3 id="syllabus-coderefinery-git-intro">Introduction to version control with Git</h3>
+    <ul>
+      <li><a href="https://coderefinery.github.io/git-intro/01-motivation/">Movitation</a></li>
+      <li><a href="https://coderefinery.github.io/git-intro/02-basics/">Basics</a></li>
+      <li><a href="https://coderefinery.github.io/git-intro/04-staging-area/">Using the Git staging area</a></li>
+      <li><a href="https://coderefinery.github.io/git-intro/05-undoing/">Undoing things</a></li>
+      <li><a href="https://coderefinery.github.io/git-intro/06-branches/">Branching and merging</a></li>
+      <li><a href="https://coderefinery.github.io/git-intro/08-conflicts/">Conflict resolution</a></li>
```
Comment (svenvanderburg) [inline _includes/cr/syllabus.html:13]: We teach this from code refinery right? Or from carpentries?

File: Gemfile [hunk 2]
```diff
@@ -5,6 +5,6 @@ source 'https://rubygems.org'
 git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
 # Synchronize with https://pages.github.com/versions
-ruby '>=2.7.1'
+ruby '>2.6'
```
Comment (svenvanderburg) [inline Gemfile:8]: I'm not a ruby-connaisseur, but this allowed me to run this locally and it doesn't seem to break stuff :)

File: _includes/navbar.html [hunk 3]
```diff
@@ -8,102 +8,94 @@
...
         <span class="sr-only">Toggle navigation</span>
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
 
-      {% comment %} Select what logo to display. {% endcomment %}
-      <a href="{{ site.esciencecenter_site }}" class="pull-left">
-        <img class="navbar-logo" src="{{ relative_root_path }}{% link /assets/img/nlesc_icon_cyanblack.png %}" alt="Software Carpentry logo" />
+      <a href="{{ site.academy_site }}" class="pull-left"></a>
```
Comment (mkuzak) [inline _includes/navbar.html]: we are not using eScience Academy branding anymore. The workshop template already includes the eScience Center logo. Please remove this?

File: index.md [hunk 4]
```diff
@@ -3,19 +3,19 @@ layout: workshop      # DON'T CHANGE THIS.
...
-enddate: FIXME        # machine-readable end date for the workshop in YYYY-MM-DD format like 2015-01-02
-instructor: ["instructor one", "instructor two"] # boxed, comma-separated list of instructors' names as strings, like ["Kay McNulty", "Betty Jennings", "Betty Snyder"]
-helper: ["helper one", "helper two"]     # boxed, comma-separated list of helpers' names, like ["Marlyn Wescoff", "Fran Bilas", "Ruth Lichterman"]
-email: ["first@example.org","second@example.org"]    # boxed, comma-separated list of contact email addresses for the host, lead instructor, or whoever else is handling questions, like ["marlyn.wescoff@example.org", "fran.bilas@example.org", "ruth.lichterman@example.org"]
+humandate: "Mar 29-Apr 1, 2021"    # human-readable dates for the workshop (e.g., "Feb 17-18, 2020")
+humantime: "9:00-13:00"    # human-readable times for the workshop (e.g., "9:00 am - 4:30 pm")
+startdate: 2021-03-29      # machine-readable start date for the workshop in YYYY-MM-DD format like 2015-01-01
+enddate: 2021-03-01        # machine-readable end date for the workshop in YYYY-MM-DD format like 2015-01-02
```
Comment (mkuzak) [inline index.md]:
```suggestion
enddate: 2021-04-01        # machine-readable end date for the workshop in YYYY-MM-DD format like 2015-01-02
```

File: index.md [hunk 5]
```diff
@@ -3,19 +3,19 @@ layout: workshop      # DON'T CHANGE THIS.
...
+instructor: ["Sven van der Burg", "Jens Wehner"] # boxed, comma-separated list of instructors' names as strings, like ["Kay McNulty", "Betty Jennings", "Betty Snyder"]
+helper: ["Alessio Sclocco", "Djura Smits", "Mateusz Kuzak"]     # boxed, comma-separated list of helpers' names, like ["Marlyn Wescoff", "Fran Bilas", "Ruth Lichterman"]
```
Comment (mkuzak) [inline index.md]:
```suggestion
helper: ["Alessio Sclocco", "Djura Smits"]     # boxed, comma-separated list of helpers' names, like ["Marlyn Wescoff", "Fran Bilas", "Ruth Lichterman"]
```

**Review by mkuzak — CHANGES_REQUESTED** [review]

File: _includes/navbar.html (hunk 3 above)
Comment (svenvanderburg) [reply _includes/navbar.html]: Check.

**Comment by svenvanderburg** [comment]
@mkuzak I implemented your suggestions, can you merge if you like it?

**Pull request merged**